backend/test/akvo/flow_services/translate_test.clj

```diff
@@ -0,0 +1,11 @@
+(ns akvo.flow-services.translate-test
+  (:require [akvo.flow-services.translate :as t]
+            [clojure.test :refer :all]))
+
+(deftest translating-strings
+  (with-redefs [t/translation-map {:some-key {:en "%s is the answer"
```

Comment:
I wonder if the important part here is the aero part. I understand that we are testing a lib (not our code), but it will check that we understand how aero works and that the format of the production file is actually correct (valid edn, valid refs)

We probably also tying to test to the production strings, but still I see the test as more useful

Reply:
When I wrote the code I was testing the _our_ code, not the aero part ... Will include another test that checks that